driver-core/src/main/com/mongodb/internal/connection/DefaultConnectionPool.java

```diff
@@ -1437,6 +1441,23 @@ public void close() {
                 maintainer.shutdownNow();
             }
         }
+
+        private void doIgnoringRejectedExecution(final Runnable action) {
```

Comment:
This method name is hard to read - "do ignoring rejected execution" doesn't read right. Can you think of a clearer name for it? `ignoreRejectedExectution` seems clearer in intent and is more readable.

Reply:
Done.